CHANGELOG.md

```diff
@@ -56,6 +56,20 @@ our [guidelines for writing a good changelog entry](https://github.com/biomejs/b
 
   Contributed by @Conaclos
 
+- Add new command `biome clean`. Use this command to purge all the logs emitted by the Biome daemon. This command is really useful, because the Biome daemon tends
+  log many files and contents during its lifecycle. This means that if your editor is open for hours (or even days), the `biome-logs` folder could become quite heavy. @Contributed by emaitpico
```

Comment:
```suggestion
  log many files and contents during its lifecycle. This means that if your editor is open for hours (or even days), the `biome-logs` folder could become quite heavy. Contributed by @emaitpico
```

Reply:
the username has a typo too

Reply:
@SuperchupuDev Your eyes are better than mines 😅